jena-extras/jena-querybuilder/src/main/java/org/apache/jena/arq/querybuilder/UpdateBuilder.java

```diff
@@ -122,6 +122,14 @@ private ExtendedIterator<Quad> getQuads(Collection<QuadHolder> holders) {
         return result;
     }
 
+    /**
+     * Checks that deletes or inserts have been added to the builder.
+     * @return true if there are any delete or insert statements.
```

Comment:
Returns true if there are no deletes and no inserts? Because the return is checking `isEmpty` for both....